bugfix: Nova Station #1 Patch

PiroMage opened the pull request:
1) Починка дверцы шкафа повара.
2) Замена плаката унати.
3) Уменьшение рейнджа звука работающего СМа.
4) Починка двигающейся статуи в брошенном храме.
5) Починка туррелей и консолей на саттелите.
6) Починка мусорки и света на аванпосте лавы.

<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание

<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР

<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений

<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

## Тесты

<!-- Как вы тестировали свой код. Ревьеюру будет проще, если он будет знать какие действия вы совершали, проверяя свой ПР. -->

PiroMage committed:
Первая волна фиксов

1) Починка дверцы шкафа повара.
2) Замена плаката унати.
3) Уменьшение рейнджа звука работающего СМа.
4) Починка двигающейся статуи в брошенном храме.
5) Починка туррелей и консолей на саттелите.
6) Починка мусорки и света на аванпосте лавы.

PiroMage committed:
Доп.Фиксы

1) Добавление переменной звука для СМа на Нове. Исправление переменной обратно для всех других СМов.
2) Обновление кода карпа для схожести с материнским кодом.
3) Обновление лавы и основной карты. Исправление недочетов.

Vladisvell reviewed (changes requested):
Error: undefined var: "zap_sound_extrarange"

PiroMage committed:
Правки в код СМ

PiroMage committed:
Изменение сма на карте после правок кода

PiroMage committed:
Мелкие фиксы

1) Перенос интеркома на лаве и добавление нового.
2) Добавление постеров на пустые стены станции.
3) наден неподключенный скраббер... Теперь подключен.
4) Визуальное улучшение проводки труб в суде.

PiroMage committed:
Фикс консоли контроля туррелей в Апплоуде

PiroMage committed:
Перенос деконструктора

1) Деконструктор рнд теперь у внутренной стойки, а не у окна.

PiroMage committed:
Улучшение карты

1) Добавление принтера в начало очереди у стойки гп.
2) Правки зон у брига.

PiroMage committed:
Еще мелкие правки

1) Правки лазиса
2) Правки станции

PiroMage committed:
Починка доступа хранилища плат

Убрана проверка одного доступа на защищенном хранилище плат.

PiroMage committed:
Замена стекол

1) Замена неломаемого стекла

Daeberdir reviewed (approved):
Подходит.
